local_planner/src/nodes/planner_functions.cpp

```diff
@@ -175,11 +156,9 @@ void propagateHistogram(Histogram &polar_histogram_est,
                         geometry_msgs::PoseStamped position) {
```

Comment:
Sorry for all the comments that are somewhat unrelated to your changes, but I just noticed this one. A `PoseStamped position` leads to the super beautiful construct `position.pose.position` down the road